### PR TITLE
Add image-util with methods for cropping images

### PR DESCRIPTION
**Pull request by astappev**
See [this](https://github.com/appium/appium-base-driver/pull/144) and [this thread](https://github.com/appium/appium-base-driver/pull/148) for more details

### CODE REVIEW

File: test/image-util-specs.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
```
Comment (mykola-mokhnach) [inline test/image-util-specs.js]: any reason to disable eslint?

**Comment by jlipps** [comment]
made #59 so we can get this merged faster.

**Comment by mykola-mokhnach** [comment]
@jlipps Do we still want to merge this PR?

**Comment by jlipps** [comment]
no, we merged #59 instead

**Pull request closed without merging**